k8s-dev/cstor-pool-mgmt/deployment-cstor-pool.yaml

```diff
@@ -0,0 +1,72 @@
+# The deployment for cstor-pool and cstor-pool-mgmt.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cstor-pool
+  labels:
+    app: cstor-pool
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cstor-pool
+  template:
+    metadata:
+      labels:
+        app: cstor-pool
+    spec:
+      containers:
+      - name: cstor-pool
+#The image name here will be changed to openebs once the business logic pr is reviewed and merged in openebs/maya.
+        image: gkganeshr/cstor-pool:v1.1
+        ports:
+        - containerPort: 80
```

Comment:
What is this port used for?

Reply:
It is normal we provide containerPort in k8s deployment yaml. In order for Docker containers to communicate across nodes, there must be allocated ports on the machine’s own IP address, which are then forwarded or proxied to the containers.

Reply:
According to official documentation containerPort is only informative. Even ports not listed here are still exposed from the container (as long as they bind to 0.0.0.0 address). I assume that nothing listens on port 80 in the container, so why would we declare the port here? Declaring port 80 here gives a false impression that there is a service listening on port 80 running inside the container. IMHO we should put here real service ports which zrepl is using and those are management, rebuild and data ports: 12000, 3233, 3232.

Reply:
will change